README.md

```diff
@@ -72,60 +80,83 @@ Notes:
 <details>
 	<summary>Expand Testing Instructions</summary>
 
-To test these configurations for the modern BPF driver on localhost using [minikube](https://minikube.sigs.k8s.io/docs/start/), make sure you have minikube installed and running. In order to test `kmod` and `bpf` drivers, additional host mounts are required. Minikube needs a specific setting to accommodate this, as shown below:
+To test these configurations for the modern BPF driver on localhost using [minikube](https://minikube.sigs.k8s.io/docs/start/), make sure you have minikube and [kubectl](https://pwittrock.github.io/docs/tasks/tools/install-kubectl/) installed and running. In order to test `kmod` and `ebpf` drivers, additional host mounts are required. Minikube needs a specific setting to accommodate this, as shown below:
 
 ```
-minikube start --mount --mount-string="/usr/src:/usr/src" --driver=docker
+minikube start --mount --mount-string="/usr/src:/usr/src" --mount --mount-string="/dev:/dev" --driver=docker --nodes 4
 ```
 
 __NOTE__: You won't be able to properly test Falco's container engine using `minikube`. Please be aware of this limitation.
 
+__NOTE__: For `localhost` testing reduce the number of replicas for the synthetic workload deployments.
+
 Proceed by executing the following setup commands:
 
-```
+```bash
 kubectl create namespace falco;
-# Always need the generic label
-kubectl label nodes minikube cncf-project=falco;
+kubectl get nodes;
 
 # Test modern-bpf (easiest)
-kubectl label nodes minikube cncf-project-sub=falco-driver-modern-bpf;
+kubectl label nodes minikube-m02 cncf-project=falco cncf-project-sub=falco-driver-modern-ebpf --overwrite;
 
 # Test bpf
-# kubectl label nodes minikube cncf-project-sub=falco-driver-bpf --overwrite;
+kubectl label nodes minikube-m03 cncf-project=falco cncf-project-sub=falco-driver-ebpf --overwrite;
 
 # Test kmod
-# kubectl label nodes minikube cncf-project-sub=falco-driver-kmod --overwrite;
+# WARNING: Testing kernel modules on a local dev box is more risky, 
+# remember to unload the module `sudo rmmod falco`
+# Testing kmod within a smaller VM with minikube likely crashes, only test w/ minikube on a larger native box
```

Comment:
Testing kmod in VMs https://github.com/falcosecurity/cncf-green-review-testing/issues/7 w/ minikube would have been too much fun ... went to my other native Linux box and it worked w/ minikube.

Note I had to modify minikube start even more. This would work after all:

```
minikube start --mount --mount-string="/usr/src:/usr/src" --mount --mount-string="/dev:/dev" --driver=docker
```